AUTHORS

```diff
@@ -44,6 +44,7 @@ Also thanks to:
     * Chris Grant (Unit158)
     * clem
     * Cody Brittain (Generalcamo)
+	* Constantin Helmig (CH4Code)

```

Comment:
Seems like we use spaces instead of tabs in this file... ^^

Reply:
Ah come on!^^